css/index.css

```
@@ -71,6 +71,22 @@ h1 {
   padding: 1em 0;
 }
 
+.policy aside {
```

```suggestion
.policy aside {
  position: sticky;
  top: 1em;
```

Feel free to reject - I just got annoyed scrolling back and forth and seems like a pretty simple change.

---

Actually this breaks really big table of contents, so ignore it.